docs/data/base/components/modal/modal.md

```diff
@@ -0,0 +1,125 @@
+---
+product: base
+title: React Modal component
+components: ModalUnstyled
+githubLabel: 'component: modal'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#dialog_modal
+---
+
+# Modal
+
+<p class="description">The `Modal` component lets you create dialogs, popovers, lightboxes, and other elements that force the user to take action before continuing.</p>
+
+The `Modal` component renders its `children` node in front of a backdrop component. This lets you create an element that your users must interact with before continuing in the parent application.
+Features:
+
+- 💄 Manages modal stacking when more than one is needed
+- 🔐 Creates a backdrop to disable interaction with the rest of the app
+- 🔐 Disables page scrolling while open
+- ♿️ Manages focus correctly between the modal and its parent app
+- ♿️ Adds the appropriate ARIA roles automatically
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+> **Note:** the term "modal" is sometimes used interchangeably with "dialog," but this is incorrect.
+> A dialog may be _modal_ or _nonmodal_.
```

Comment:
```suggestion
> A dialog may be _modal_ or _modeless_.
```

Reply:
My impression is that these two words are interchangeable. I'm not married to either one. [Modal & Nonmodal Dialogs](https://www.nngroup.com/articles/modal-nonmodal-dialog/) was one of my sources here.

Reply:
As the two terms are used in different places, how about we do:

```suggestion
> A dialog may be _modal_ or _nonmodal (modeless)_.
```

Reply:
Perfect! :)